### PR TITLE
Set a bunch of project encodings to UTF-8

### PR DESCRIPTION
**Pull request by fedejeanne**
It was done using a "Quick Fix" and the value set (UTF-8) is the same value that any other project in the workspace uses. The objective of this commit is to declutter the "Problems" view.

### CODE REVIEW

File: org.eclipse.jdt.ui.unittest.junit/.settings/.api_filters [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.unittest.junit/.settings/.api_filters:1]: Unrelated change, please revert.

File: org.eclipse.jdt.text.tests/testResources/folderLinkTarget1/.settings/org.eclipse.core.resources.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.text.tests/testResources/folderLinkTarget1/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Reference/.settings/org.eclipse.core.resources.prefs [hunk 4]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Reference/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Source/.settings/org.eclipse.core.resources.prefs [hunk 5]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Source/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/ExtractMethodWorkSpace/ExtractMethodTests/.settings/org.eclipse.core.resources.prefs [hunk 6]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/ExtractMethodWorkSpace/ExtractMethodTests/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/InlineMethodWorkspace/TestCases/.settings/org.eclipse.core.resources.prefs [hunk 7]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/InlineMethodWorkspace/TestCases/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/ReplaceInvocationsWorkspace/ReplaceInvocations/.settings/org.eclipse.core.resources.prefs [hunk 8]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/ReplaceInvocationsWorkspace/ReplaceInvocations/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/SefWorkSpace/SefTests/.settings/org.eclipse.core.resources.prefs [hunk 9]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/SefWorkSpace/SefTests/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/SurroundWithWorkSpace/SurroundWithTests/.settings/org.eclipse.core.resources.prefs [hunk 10]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/SurroundWithWorkSpace/SurroundWithTests/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests.refactoring/resources/TypeEnvironment/TestProject/.settings/org.eclipse.core.resources.prefs [hunk 11]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests.refactoring/resources/TypeEnvironment/TestProject/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests/testresources/JUnitWorkspace/JUnit4Tests/.settings/org.eclipse.core.resources.prefs [hunk 12]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests/testresources/JUnitWorkspace/JUnit4Tests/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

File: org.eclipse.jdt.ui.tests/testresources/JUnitWorkspace/JUnitTests/.settings/org.eclipse.core.resources.prefs [hunk 13]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (iloveeclipse) [inline org.eclipse.jdt.ui.tests/testresources/JUnitWorkspace/JUnitTests/.settings/org.eclipse.core.resources.prefs:1]: This is a test folder, why it is changed?

**Review by iloveeclipse — CHANGES_REQUESTED** [review]

File: org.eclipse.jdt.ui.unittest.junit/.settings/.api_filters (hunk 1 above)
Comment (fedejeanne) [reply org.eclipse.jdt.ui.unittest.junit/.settings/.api_filters]: I reverted the unwanted commit. Thank you!

File: org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs (hunk 3 above)
Comment (fedejeanne) [reply org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs:1]: For what I see in [`ValidateProjectEncoding`](https://github.com/eclipse-platform/eclipse.platform/blob/master/resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/ValidateProjectEncoding.java#L76C31-L76C31), the validation is applied in all projects in the workspace without making any distintion as to whether they are test projects or not.

Do you think these changes should be reverted? I think they don't really bother much and the warning just clutters the _Problems_ view, that's why I applied the _Quick Fix_ to all of them.

Comment (iloveeclipse) [reply org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs:1]: The test projects modified here serve the purpose to be used in tests. If there are no test fails, why is the change needed? Usually, none of the projects here are imported in any development workspace.

Comment (fedejeanne) [reply org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs:1]: > If there are no test fails, why is the change needed? 

The idea is to declutter the Problems view and this is a low hanging fruit: it can be done easily with a Quick Fix and AFAICT it doesn't really break anything.

> Usually, none of the projects here are imported in any development workspace.

I installed my IDE following [the official instructions in Contributing.md](https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md#creating-an-eclipse-development-environment) and I got all those projects checked out, _e.g._ these JDT tests
grafik

Comment (iloveeclipse) [reply org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs:1]: @merks : could you please check jdt ui setup? I assume it misses some filters.

**Comment by fedejeanne** [comment]
All these projects are test projects and should not be changed. I'm withdrawing this PR.
Thank you @iloveeclipse for reviewing!

File: org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Source/.settings/org.eclipse.core.resources.prefs (hunk 5 above)
Comment (merks) [reply org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Source/.settings/org.eclipse.core.resources.prefs:1]: Please open an issue and assign it to me so that I/we don’t forget.

Comment (fedejeanne) [reply org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/Source/.settings/org.eclipse.core.resources.prefs:1]: See https://github.com/eclipse-jdt/eclipse.jdt.ui/issues/903
